### DISCUSSION TITLE
Şanlı kullanıcı 60.000 ibax hakkında..

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#1227] [Soru Cevap - Türkçe] [answered]
3. Şanslı Kullanıcı Ödülü
Kampanyanın 60 günü boyunca, 50.000 IBXC almak için her gün rastgele bir şanslı kullanıcı seçilecektir (Ödül tekrar tekrar kazanılamaz)

**Comment by Abdullahoda** [comment]
Bu ödülü kazanan duymadık açıklıyanda olmadı sanırım ödülü vermekten vazgeçtiler.

**Reply by Yusufknk** [reply]
Ödüller için düzenleme yapılıyor

**Comment by erhanqwerty** [comment]
must wait, please

**Comment by sedat2855** [comment]
DAHA ACIKLANMADI SANIRIM

**Comment by Bulent3407** [comment, marked as answer]
Ödüller verilecektir 1 şubatta
